example-service/src/client.rs

```diff
@@ -6,8 +6,8 @@
 
 use clap::{App, Arg};
 use std::{io, net::SocketAddr};
+use tarpc::tokio_serde::formats::Json;
```

Comment:
nit: put this in the `use tarpc::{ .. }` block below.

Reply:
Both done :)